### PR TITLE
Update react

### PR DESCRIPTION
**Pull request by ecgeiser**

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,46 @@
+0 info it worked if it ends with ok
```
Comment (rovolution) [inline npm-debug.log]: remove this file from the PR

File: package.json [hunk 2]
```diff
@@ -18,7 +19,9 @@
   },
   "jest": {
     "scriptPreprocessor": "preprocessor.js",
-    "unmockedModulePathPatterns": ["node_modules/react"]
```
Comment (rovolution) [inline package.json:21]: we need to add a `prepublish` script to the `"scripts"` property that runs babel on the source code

That will run babel and generate the `/lib` contents when we type `npm publish`

Comment (ecgeiser) [inline package.json:21]: added the `prepublish` script - can you run it locally as a sanity check?

File: .gitignore [hunk 3]
```diff
@@ -1 +1,4 @@
 node_modules
+.DS_Store
+lib
+.npm-debug.log
```
Comment (rovolution) [inline .gitignore]: filename is `npm-debug.log` without the `.` in front

Comment (ecgeiser) [inline .gitignore]: thanks, good catch

File: package.json (hunk 2 above)
Comment (rovolution) [inline package.json:21]: tried running it and kept getting this error:

image

**Pull request merged**